README.md

```diff
@@ -1,26 +1,60 @@
-# Offscreen Effect Player module
+# [Offscreen Effect Player module (OEP-module)](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview#offscreen-effect-player)
```

Comment:
If you give the same link in the body of the topic then probably it is not necessary to duplicate it in the header.